app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,9 @@
         tools:targetApi="31">
         <activity
             android:name=".MainActivity"
-            android:exported="true">
+            android:exported="true"
+            android:windowSoftInputMode="adjustResize"
```

Comment:
we don't need to set android:windowSoftInputMode, do we ? 🤔

Reply:
We don't need this.
This was added to support padding for software keyboard in chat screen.
But `imePadding()` seems to do the work.